samples/sampler/stories/qa/VirtualList.js

```diff
@@ -545,13 +545,13 @@ WithContainerItemsHaveSpottableControls.parameters = {
 
 const fixedItemSizes = new Array(16).fill(ri.scale(390));
 const variableItemSizes = fixedItemSizes.map((size, index) => {
-	return index % 2  ? size * 2  : size;
+	return index % 2 ? size * 2 : size;
 });
 
 // eslint-disable-next-line enact/prop-types, enact/display-name
-const renderVirtualListItem = (data) => ({index, ...rest}) => {
+const renderVirtualListItem = (data, onClick = {}) => ({index, ...rest}) => {
 	return (
-		<Item {...rest} style={{width:data[index], margin: ri.scaleToRem(15)}}>
+		<Item {...rest} style={{width: data[index], margin: ri.scaleToRem(15)}} onClick={onClick(index)}>
```

Comment:
Related to onClick changes, as `renderVirtualListItem` is used in 'with changing item sizes' and 'with changing fixed and variable item sizes', so these samples also need to be fixed. There are errors in both samples.

Reply:
I didn't mean to use `{}` as default value for `onClick`. 😄

Reply:
Looks good to me